extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -103,7 +106,24 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 				width = newWidth;
```

Comment:
what params will be passed to this function in case we want to resize to fullscreen?

Reply:
nothing, it's obtained automatically in fullscreen mode https://github.com/Wikia/app/pull/14227/files#diff-e77e2c1ef4d0877d80d59d95c1086bc9R153
but we still want to update these values during resizing.